docs/en/connector-v2/sink/Doris.md

```diff
@@ -17,67 +30,97 @@ Version Supported
+| sink.buffer-size    | int    | No       | 256 * 1024 | the buffer size to cache data for stream load.                                                                                                                                                                                                                                               |
+| sink.buffer-count   | int    | No       | 3          | the buffer count to cache data for stream load.                                                                                                                                                                                                                                              |
+| doris.config        | map    | yes      | -          | This option is used to support operations such as `insert`, `delete`, and `update` when automatically generate sql,and supported formats.                                                                                                                                                    |
 
-`Doris` user password
-
-### table.identifier [string]
-
-The name of `Doris` table
-
-### sink.label-prefix [string]
-
-The label prefix used by stream load imports. In the 2pc scenario, global uniqueness is required to ensure the EOS semantics of SeaTunnel.
-
-### sink.enable-2pc [bool]
-
-Whether to enable two-phase commit (2pc), the default is true, to ensure Exactly-Once semantics. For two-phase commit, please refer to [here](https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD).
-
-### sink.enable-delete [bool]
+#### Supported import data formats
 
-Whether to enable deletion. This option requires Doris table to enable batch delete function (0.15+ version is enabled by default), and only supports Unique model. you can get more detail at this link:
+The supported formats include CSV and JSON
 
-https://doris.apache.org/docs/dev/data-operate/update-delete/batch-delete-manual
+## Task Example
 
-### doris.config [map]
+### Simple:
 
-The parameter of the stream load `data_desc`, you can get more detail at this link:
+> This example defines a SeaTunnel synchronization task that automatically generates data through FakeSource and sends it to Doris Sink,FakeSource simulates CDC data with schema pk_id (bigint type), name (string type), and score (int type),Doris needs to create a table sink named test.e2e_table_sink and a corresponding table for it.
 
-https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD
+```hocon
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+  checkpoint.interval = 10000
+}
 
-#### Supported import data formats
+source {
+  FakeSource {
+    schema = {
+      fields {
+        pk_id = bigint
```

Comment:
Add all the data types, it's more friendly

Reply:
I have added as many types as possible that are supported by all versions